README.md

```diff
@@ -123,6 +123,22 @@ __Properties__
 * Average case performance	O(log n)
 * Worst case space complexity	O(1) 
 
+### Interpolation
+Interpolation search is an improved version of binary search algorithm.
+
+![alt text](https://qph.ec.quoracdn.net/main-qimg-02f1f050de01608b9b1f2f27155d1b17)
+
+Even when the data is sorted, binary search does not take advantage of that to probe the position of desired data.
+Position Probing in Interpolation Search
+Interpolation search search a particular item by computing the probe position. Initially probe position is the position of the middle most item of the collection.
+If middle item is greater than item then probe position is again calculated in the sub-array to the right of the middle item other wise item is search in sub-array to the left of the middle item. This process continues on sub-array as well until the size of subarray reduces to zero.
+
+__Properties__
```

Comment:
use ##

Reply:
in place of ### or somewhere else

Reply:
well using ## will make it more bigger and then it might look different then other can u review it one time pls then tell me what to do

Reply:
in place of __Properties__

Reply:
ok i have to remove properties and make the underwriting bigger or i have to make properties size bigger

Reply:
properties size

Reply:
and if you could include a gif for this algo like the other ones in the readme?